composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": "^8.0",
+        "illuminate/support": "^9.7",
```

Comment:
It's always better to lock the minimum supported version either using `illuminate/support` or `illuminate/contract` under `require`